doc/source/biascorr.md

```diff
@@ -15,146 +17,420 @@ kernelspec:
+:mystnb:
+:  code_prompt_show: "Show plotting code"
+:  code_prompt_hide: "Hide plotting code"
+
+# Plot before and after
+f, ax = plt.subplots(1, 2)
+ax[0].set_title("Before directional\nde-biasing")
+(tbc_dem_sumsin - ref_dem).plot(cmap='RdYlBu', vmin=-30, vmax=30, ax=ax[0])
+ax[1].set_title("After directional\nde-biasing")
+(corrected_dem - ref_dem).plot(cmap='RdYlBu', vmin=-30, vmax=30, ax=ax[1], cbar_title="Elevation differences (m)")
+_ = ax[1].set_yticklabels([])
+```
+
+For strip-like errors, performing an empirical correction using only a binning with `fit_or_bin="bin"` allows more
+flexibility, but requires a larger amount of static surfaces.
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+:mystnb:
+:  code_prompt_show: "Show the code for adding a strip bias"
+:  code_prompt_hide: "Hide the code for adding a strip bias"
+
+# Get rotated coordinates along an angle
+angle = 60
+xx = gu.raster.get_xy_rotated(ref_dem, along_track_angle=angle)[0]
+
+# Create a strip bias and add to the DEM
+synthetic_bias = np.zeros(np.shape(ref_dem.data))
+xmin = np.min(xx)
+synthetic_bias[np.logical_and((xx - xmin)<1200, (xx - xmin)>800)] = 20
+synthetic_bias[np.logical_and((xx - xmin)<2800, (xx - xmin)>2500)] = -10
+synthetic_bias[np.logical_and((xx - xmin)<5300, (xx - xmin)>5000)] = 10
+synthetic_bias[np.logical_and((xx - xmin)<15000, (xx - xmin)>14500)] = 5
+synthetic_bias[np.logical_and((xx - xmin)<21000, (xx - xmin)>20000)] = -15
+tbc_dem_strip = ref_dem + synthetic_bias
+```
 
 ```{code-cell} ipython3
-# Instantiate a directional bias correction
-dirbias = coreg.DirectionalBias(angle=65)
-# Fit the data
-dirbias.fit(ref_dem, tba_dem, inlier_mask=inlier_mask)
+# Define a directional bias correction at a certain angle (degrees), for a binning of 1000 bins
+dirbias = xdem.coreg.DirectionalBias(angle=60, fit_or_bin="bin", bin_sizes=1000)
+# Fit and apply
+corrected_dem = dirbias.fit_and_apply(ref_dem, tbc_dem_strip)
+```
 
-# Apply the transformation
-corrected_dem = dirbias.apply(tba_dem)
+```{code-cell} ipython3
+:tags: [hide-input]
+:mystnb:
+:  code_prompt_show: "Show plotting code"
+:  code_prompt_hide: "Hide plotting code"
+
+# Plot before and after
+f, ax = plt.subplots(1, 2)
+ax[0].set_title("Before directional\nde-biasing")
+(tbc_dem_strip - ref_dem).plot(cmap='RdYlBu', vmin=-30, vmax=30, ax=ax[0])
+ax[1].set_title("After directional\nde-biasing")
+(corrected_dem - ref_dem).plot(cmap='RdYlBu', vmin=-30, vmax=30, ax=ax[1], cbar_title="Elevation differences (m)")
+_ = ax[1].set_yticklabels([])
 ```
 
-## Terrain biases
+### Terrain biases
 
 {class}`xdem.coreg.TerrainBias`
 
-- **Performs:** Correct biases along a terrain attribute of the DEM.
-- **Supports weights** Yes.
-- **Recommended for:** Different native resolution between DEMs.
+- **Performs:** Correct biases along a terrain attribute.
+- **Supports weights:** Yes.
+- **Pros:** Useful to correct for instance curvature bias due to different native resolution between elevation data.
```

Comment:
in Arnaud's project we ended up calling it the "resolution bias" as curvature is not the best proxy

Reply:
Good point. Hard to change the name here as I add a bias based on curvature in the example below, we can adjust once we have the TPI implementation :wink: